### DISCUSSION TITLE
i can't import my history and playlists because google takeout exported them respectively in .html and .csv and freetube wants a "Database File"

### DISCUSSION THREAD

**Discussion by Nekothegamer** [FreeTubeApp/FreeTube#2521] [Q&A]
the title say it, i can't import those stuff because freetube needs a .db file (i suppose) and as far as i know it's impossible to convert a .csv into .db, so help me find a way to resolve this problem please

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
These are known issues
For playlists see https://github.com/FreeTubeApp/FreeTube/issues/1016. 
For history see #1623.

No users have reported workarounds
